### DISCUSSION TITLE
Indent Animation: How to disable?

### DISCUSSION THREAD

**Discussion by dijeferson** [folke/snacks.nvim#281] [Q&A]
I have been trying to disable the animations for indent and it simply won't stick to it. Am I doing it wrong?

```lua
return {
  "folke/snacks.nvim",
  priority = 1000,
  lazy = false,
  opts = {
    bigfile = { enabled = true },
    dashboard = { enabled = true },
    indent = { enabled = true, animation = { enabled = false } },
    input = { enabled = true },
    notifier = { enabled = true },
    quickfile = { enabled = true },
    scroll = { enabled = true },
    statuscolumn = { enabled = true },
    words = { enabled = true },
  },
}
```

Thanks!

**Comment by folke** [comment]
Yes you are doing it wrong. Did you check the actual documentation for `indent`?
If so you would have noticed it should be `animate` instead of  `animation`

**Reply by dijeferson** [reply]
Thanks for pointing it out. Yesterday I got to the point of actually copying and pasting the default configs from the docs to try to disable it and it didn't work. 

I rolled back my code (to use the defaults + enabled=false) and updated snacks.nvim to the latest version and it seems to have solved the issue. 

Thank you
